app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_file = command.split()
```

Comment:
Code style issue: The variable name 'list_file' is not descriptive. It would be better to use a more descriptive name, like 'command_parts' or 'command_elements'.